src/index.js

```diff
@@ -1,19 +1,42 @@
-import _ from 'lodash';
-import printMe from './print.js';
 import './style.css';
 
- function component() {
-   const element = document.createElement('div');
-  const btn = document.createElement('button');
+const tasks = [
+  {
+    index: 1,
+    description: 'Task 1',
+    Complete: true,
+  },
+  {
+    index: 2,
+    description: 'Task 2',
+    Complete: false,
+  },
+  {
+    index: 3,
+    description: 'Task 3',
+    Complete: false,
+  },
+];
 
-   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+function createListItem(task) {
+  return `
+    <li ${task.Complete ? 'class="complete"' : ''}>
+      <div>
+        <input type="checkbox" name="tasks" ${
+  task.Complete ? 'checked' : ''
+} id="${task.index}" />
+        <span>${task.description}</span>
+      </div>
+      <button title="Remove task" class="remove-task">
+        <svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 128 512"><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. --><path d="M64 360a56 56 0 1 0 0 112 56 56 0 1 0 0-112zm0-160a56 56 0 1 0 0 112 56 56 0 1 0 0-112zM120 96A56 56 0 1 0 8 96a56 56 0 1 0 112 0z"/></svg>
+      </button>
+    </li>
+  `;
+}
 
-  btn.innerHTML = 'Click me and check the console!';
-  btn.onclick = printMe;
+function renderList() {
+  const todosListItems = document.getElementById('todos');
+  todosListItems.innerHTML = tasks.map(createListItem).join('');
+}
 
-  element.appendChild(btn);
-
-   return element;
- }
-
- document.body.appendChild(component());
\ No newline at end of file
+renderList();
```

Comment:
- [OPTIONAL] You have done great meeting the project requirement :+1: :+1: I'll suggest calling this function only when the DOM content is fully loaded, this will make sure that all the HTML is present on the page before appending the list with JS. You can achieve this by passing the `renderList` function as a callback to a dom content-loaded event listener. :+1: :100: 

```sh
document.addEventListener("DOMContentLoaded", () => renderList());
```

Reply:
@V-Blaze 
Your suggestion to call the `renderList` function only when the DOM content is fully loaded is fantastic. Not only does it align perfectly with the project requirements, but it also showcases your deep understanding of best practices. Your guidance to attach the function as a callback to the DOM content-loaded event listener is an elegant solution that ensures seamless interaction between our HTML and JavaScript components.